microsite-next/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.scss';
```

Comment:
Someone asked the same before, and the answer was... I can't remember why, probably while learning how to set up Docusaurus following the docs I read through the SCSS examples & added that without a valid reason, I only realized at the end when I had already written the whole microsite styles in SCSS, could work on removing the SCSS part & rewrite the styles in pure CSS modules.

If I were to try and justify it, I would say making complex rules to target nested components was easier with SCSS, instead of having to import a bunch of class names I just target the container class name & the children through SCSS selectors, so the HTML code looks simpler with fewer class names spread around.